src/content/pages/explore.mdx

```diff
@@ -6,136 +6,129 @@ subtitle: At EuroPython, many attendees will come with their families. While not
+- If you want to visit other towns, trains are a convenient and economical option.
 
 ### Be prepared for the weather
 
-If you want to explore the city, you will be exposed to the weather. So here are some answers to frequent questions like: What’s the weather like? What should you carry with you?
+If you want to explore the city, you should be prepared for the weather. Here are answers to some common questions:
 
-- Usually 30°C+, so carry water
-- There are no water fountains or public toilets, so carry more water
-- Use sunscreen
-- It rarely rains, but when it does, it’s shower rain. It’s best to just hide somewhere until it stops (it usually stops after half an hour or so).
+- Temperatures are usually above 30°C, so remember to carry water.
+- Public water fountains (and toilets) are rare, so remember to carry more water.
+- Use sunscreen.
+- It rarely rains, but when it does, it tends to be brief showers. Find shelter until it passes, usually within about half an hour.
 
 ## What to do while in Prague?
 
-There are plenty of things you can do in Prague. Depending on your preferences, you can find things that interest you and your group.
+There are plenty of things to enjoy in Prague. Depending on your preferences, you can find things that interest you and your group.
 
-### For culture lovers
+### For culture and art lovers
 
-- Summer cinema
-- Theater festival Za dveřmi 2024
-
-### For art lovers
-
-- DOX
-- Soví mlýny
-- National Museum
-- Jazz Dock for live jazz on a boat
-- Žižkov area for bars and pubs
-- House of the Black Madonna: cafe shop in a building that’s built in cubist style (there was a cubist architecture movement in Prague and it’s quite famous and unique)
+- Summer cinema (Check some available options [here](https://www.praguest.com/en/outdoor-cinemas-in-prague))
+- Theater festival [Za dveřmi 2024](https://zadvermi.cz/)
+- [DOX](https://www.dox.cz/en)
+- [Museum Kampa](https://www.museumkampa.cz/en/)
+- [National Museum](https://www.nm.cz/en)
+- [House of the Black Madonna](https://en.wikipedia.org/wiki/House_of_the_Black_Madonna): café in a cubist-style building (an early example Prague's famous and unique cubist architecture movement)
 
 ### For nature lovers
 
-- Petřín Hill: Eiffel Tower (in Prague), nice view of the city
-- Vyšehrad: nice fortress next to the venue. It also features a park where you can sit and chill
-- Letná: viewpoint
-- Riegrovy Sady: park with a nice view. There’s a rose garden most people don’t know about - you should check it out
-- Divoká Šárka: small forest a bit further away, you can swim in Džbán (an artificial lake)
-- Kunratický Les: forest
-- Hostivář: area for walking, you can swim there as well
+- Petřín Hill: Prague's version of the Eiffel Tower, offering a scenic city view
+- Vyšehrad: Fortress next to the venue, also featuring a park that is ideal for relaxation
+- Riegrovy Sady: Park with a gorgeous view. Remember to check out the hidden rose garden!
```

Comment:
@clytaemnestra are you sure there is a rose garden here, and not in Petřín park?

Reply:
it's hidden on the map! not even locals know about it, but I studied there in the area, so I spent quite some time there. it's basically here: https://www.google.com/maps/@50.0816828,14.4422925,3a,75y,15.73h,69.37t/data=!3m6!1e1!3m4!1sAF1QipOY1yHLaYEfHY7dzqs-6SEKCNsEfVKNdu2uwUUt!2e10!7i7680!8i3840?coh=205409&entry=ttu , but it doesn't look nice in winter, because there are no 🌹🌹🌹.

Reply:
there's a rose garden in Petřín as well, I think we can mention it - it's very nice there. there's a nice botanical garden in Troja as well, but not sure if people are interested in it.

Reply:
maybe also we could include info about canoeing, kayaking and padleboards. this is what I wrote a few days ago, feel free to reuse and adopt if it makes sense: 
> You can also rent a paddleboard, but I'd recommend the Troja area. Vltava is very slow and chill, but if you want something more exciting, there are rapids for kayaking as well. There's a 10km long cycle path next to the river all the way to Roztoky. In Roztoky, there's the únětice Brewery, which is my favorite brewery - but I recommend not grabbing a beer there, but at U Lasíků, which is a nice café next to it. You can then come back to Prague by train - it costs a few euros and it's very fast. Alternatively, you can take a ferry and go to Stromovka Park on the other side and take a tram/metro back. 
One more tip: canoeing and camping are Czech national sports—everyone does it and camps are full every summer. For hiking, canoeing, kayaking, and similar activities, I totally recommend mapy.cz. It works even abroad and it's way better than google maps when in nature. The app marks hiking routes (red, green, and blue), camps, and everything you need. You can also plan your trip on the river and see what to visit on your way, for example: [https://en.mapy.cz/zakladni?planovani-trasy&rc=9gxyAxXP6piE-xXrZV&rs=base&rs=pubt&ri=2117079&ri=18926195&mrp=%7B"c"%3A143%7D&xc=%5B%5D&rwp=1%3B9gy4lxX-kK5P4xXkO4&x=14.4093291&y=50.0660145&z=15](https://en.mapy.cz/zakladni?planovani-trasy&rc=9gxyAxXP6piE-xXrZV&rs=base&rs=pubt&ri=2117079&ri=18926195&mrp=%7B%22c%22%3A143%7D&xc=%5B%5D&rwp=1%3B9gy4lxX-kK5P4xXkO4&x=14.4093291&y=50.0660145&z=15)

Reply:
I will try to add them them all tomorrow! Thanks!

Reply:
here https://github.com/EuroPython/website/pull/748/commits/d5170535912668ad3f56ff2015c1ed834a94e3c7 
- I added the two Rose gardens & the Botanic Garden in the `For nature lovers` section. 
- in a new section called `For sports and outdoor enthusiasts`, I tried my best to add all your activity suggestions. Can you double check? I don't know what I am writing since I am not familiar with outdoor things :sweat_smile:

Reply:
amazing, thank you so much! 😍

Reply:
Thank you for all the great suggestions! Made me even consider visiting a forest for once! <3